ftdetect/markdown.vim

```diff
@@ -1,3 +1,3 @@
 " markdown filetype file
-au BufRead,BufNewFile *.{md,mdown,mkd,mkdn,markdown,mdwn}   set filetype=mkd.markdown
```

Comment:
Then you have to reserve those 'mkd.xxx' with those new 'markdown.xxx' files. 'set ft=mkd.markdown' won't work because vim cannot find 'mkd.vim' files